src/date-util/date-util.ts

```diff
@@ -120,6 +120,21 @@ export namespace DateUtil {
     }
     return min;
   }
+
+  export function secondsToTimeFormat(seconds: number): string {
+    if (seconds < 0) {
+      throw new Error('Invalid number');
+    }
+
+    const hh = Math.floor(seconds / 3600)
+      .toString()
+      .padStart(2, '0');
+    const mm = Math.floor((seconds % 3600) / 60)
+      .toString()
+      .padStart(2, '0');
+    const ss = (seconds % 60).toString().padStart(2, '0');
+    return `${hh}:${mm}:${ss}`;
```

Comment:
개인적으로 ceil, floor, round 등을 안쓰는 방법을 선호해서 제안해봅니다.
무시하셔도 됩니다.

```suggestion
    const ss = seconds % 60;
    let minutes = 0,
      mm = 0,
      hh = 0;

    if (seconds > 60) {
      minutes = (seconds - ss) / 60;
      mm = minutes % 60;

      if (minutes > 60) {
        hh = (minutes - mm) / 60;
      }
    }

    return `${hh.toString().padStart(2, '0')}:${mm.toString().padStart(2, '0')}:${ss.toString().padStart(2, '0')}`;
```

Reply:
`parseInt` 를 쓰는건 어떨까요?

`const hh = parseInt(seconds / 3600)
	      .toString()
	      .padStart(2, '0');`

Reply:
제 코드의 내용은 나머지를 빼게 되면 60의 약수가 되니까 소수점을 생각할 필요가 없다입니다.
그러면서 seconds가 작으면 불필요한 연산도 하지 않겠다는 것이죠.
parseInt()나 Math.floor()나 소수점 이하가 생기면 잘라주는 거라서 원 코드와 별 차이가 없습니다.

다만 웬만하면 소수점 이하 숫자가 안생기게 하려는 저의 편집증일 뿐
제 코드가 특별히 성능상 이점은 없으므로 그냥 원래대로 해도 됩니다.